src/scrape_up/twitter/numidconverter.py

```diff
@@ -0,0 +1,61 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+from time import sleep
+from bs4 import BeautifulSoup
+import json
+
+def unametoid(username):
+    url = 'https://twitter.com/{}'.format(username)
```

Comment:
Hey @GayathriManeksha | The code-base looks good, but please follow the folder structure. 
+ Add `__init__.py` file in the `scrape_up/twitter` directory.
+ You can see the `scrape_up/twitter` directory for reference.
+ Also add the method and class in the `./documentation.md` file.